presto-orc/src/main/java/com/facebook/presto/orc/stream/BooleanInputStream.java

```diff
@@ -208,22 +208,22 @@ public void getSetBits(Type type, int batchSize, BlockBuilder builder)
     public void getSetBits(Type type, int batchSize, BlockBuilder builder, boolean[] isNull)
             throws IOException
     {
-        for (int i = 0; i < batchSize; i++) {
+        getSetBits(type, batchSize, builder, isNull, 0);
+    }
+
+    /**
+     * Sets the vector element to true for the batchSize number of elements starting at offset
```

Comment:
javadoc is wrong. There is no `vector` here.

Reply:
I'll update this in the next refactoring PR.